libs/@guardian/source-foundations/src/tokens.test.ts

```diff
@@ -903,6 +904,151 @@ describe('Typography API object style output', () => {
+			font-size: 3.125rem;
+			line-height: 1.15;
+			font-weight: 700;
+			font-style: normal;
+		`,
+			{ isFragment: true },
+		);
+		expect(presets.headlineLight50).toMatchCSS(
+			`
+			font-family: "GH Guardian Headline", "Guardian Egyptian Web", Georgia, serif;
+			font-size: 3.125rem;
+			line-height: 1.15;
+			font-weight: 300;
+			font-style: normal;
+		`,
+			{ isFragment: true },
+		);
+		expect(presets.headlineLightItalic50).toMatchCSS(
+			`
+			font-family: "GH Guardian Headline", "Guardian Egyptian Web", Georgia, serif;
+			font-size: 3.125rem;
+			line-height: 1.15;
+			font-weight: 300;
+			font-style: italic;
+		`,
+			{ isFragment: true },
+		);
+		expect(presets.kicker20).toMatchCSS(
+			`
+			font-family: "GH Guardian Headline", "Guardian Egyptian Web", Georgia, serif;
+			font-size: 1.25rem;
+			line-height: 1.15;
+			font-weight: 700;
+			font-style: normal;
+		`,
+			{ isFragment: true },
+		);
+		expect(presets.textEgyptian17).toMatchCSS(
+			`
+			font-family: GuardianTextEgyptian, "Guardian Text Egyptian Web", Georgia, serif;
+			font-size: 1.0625rem;
+			line-height: 1.3;
+			font-weight: 400;
+			font-style: normal;
+		`,
+			{ isFragment: true },
+		);
+		expect(presets.textEgyptianBold17).toMatchCSS(
+			`
+			font-family: GuardianTextEgyptian, "Guardian Text Egyptian Web", Georgia, serif;
+			font-size: 1.0625rem;
+			line-height: 1.3;
+			font-weight: 700;
+			font-style: normal;
+		`,
+			{ isFragment: true },
+		);
+		expect(presets.textEgyptianBoldItalic17).toMatchCSS(
+			`
+			font-family: GuardianTextEgyptian, "Guardian Text Egyptian Web", Georgia, serif;
+			font-size: 1.0625rem;
+			line-height: 1.3;
+			font-weight: 700;
+			font-style: italic;
+		`,
+			{ isFragment: true },
+		);
+		expect(presets.textSans12).toMatchCSS(
+			`
+			font-family: GuardianTextSans, "Guardian Text Sans Web", "Helvetica Neue", Helvetica, Arial, "Lucida Grande", sans-serif;
+			font-size: 0.75rem;
+			line-height: 1.3;
+			font-weight: 400;
+			font-style: normal;
+		`,
+			{ isFragment: true },
+		);
+		expect(presets.textSansBold12).toMatchCSS(
+			`
+			font-family: GuardianTextSans, "Guardian Text Sans Web", "Helvetica Neue", Helvetica, Arial, "Lucida Grande", sans-serif;
+			font-size: 0.75rem;
+			line-height: 1.3;
+			font-weight: 700;
+			font-style: normal;
+		`,
+			{ isFragment: true },
+		);
+		expect(presets.textSansItalic12).toMatchCSS(
+			`
+			font-family: GuardianTextSans, "Guardian Text Sans Web", "Helvetica Neue", Helvetica, Arial, "Lucida Grande", sans-serif;
+			font-size: 0.75rem;
+			line-height: 1.3;
+			font-weight: 400;
+			font-style: italic;
+		`,
+			{ isFragment: true },
+		);
+		expect(presets.titlepiece42).toMatchCSS(
+			`
+			font-family: "GT Guardian Titlepiece", Georgia, serif;
+			font-size: 2.625rem;
+			line-height: 1.15;
+			font-weight: 700;
+			font-style: normal;
+		`,
+			{ isFragment: true },
+		);
+	});
+});
```

Comment:
Were we going to add all of the presets to this test?

Reply:
I wasn't sure if it was worth the effort to create tests for all 97 presets in case the output changes as this is still being worked on, but realised I can automate creating these by repurposing the build script so have added a complete set now.